syft/__init__.py

```diff
@@ -3,12 +3,17 @@
 PySyft decouples private data from model training, using Federated Learning,
 Differential Privacy, and Multi-Party Computation (MPC) within PyTorch.
 """
+# We load this module first so that syft knows which frameworks are available
+from syft import dependency_check
+
 # Major imports
 from syft import frameworks
-from syft import workers
 from syft import codes
-from syft import federated
-from .version import __version__
+from syft.version import __version__
+
+# This import statement is strictly here to trigger registration of syft
+# tensor types inside hook_args.py.
+import syft.frameworks.torch.hook.hook_args
```

Comment:
I understand why you did this, but I wonder if there is not a more pythonic way to trigger this registration?

Reply:
open to ideas!